<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>차트패턴검색기</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>15</y>
           <width>56</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>기간설정:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineDayFrom">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>10</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>20100101</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineDayTo">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>10</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>20100101</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>135</x>
           <y>15</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>~</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkModified">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>13</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>수정주가</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textCondition">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>110</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="0">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="0">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>240</red>
              <green>240</green>
              <blue>240</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="plainText">
         <string>유의사항
 1. 데이터는 특정한 날을 기준으로 수집되어, 상장폐지, 유무상증자로 인한 권리락 등의 이유로 일부 누락과 오류가 있을 수 있음
 2. 수정주가는 일봉(x+1)['prev'] / 일봉(x)['close'] 차이를 이용하여 간접적으로 마지막 날부터 시작날까지 역순으로 구하는 방법을 사용하며, 따라서 검색일 당일 실제 수정주가와는 가격에 차이가 발생할 수 있음.
 3. 데이터나 프로그램상의 오류로 인하여 실제와 다른 통계 결과가 나올 수 있으므로 반드시 모의투자 등을 이용하여 검증을 할 것을 권장하며, 프로그램은 투자에 있어서 참고 용도로만 사용해야함. 잘못된 통계적 결과물에 대해 개발자는 어떠한 책임도 지지 않음.
 4. 이 프로그램을 참고하여 발생한 투자 손실에 대하여 개발자는 어떠한 책임도 지지 않음.</string>
        </property>
        <property name="backgroundVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://github.com/attainer/StockChartAnalyst&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;프로그램 정보 및 도움말&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="buttonOpenCondition">
        <property name="text">
         <string>불러오기</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSaveCondition">
        <property name="text">
         <string>조건 저장</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonRunCondition">
        <property name="text">
         <string>조건 실행</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonOpenChart">
        <property name="text">
         <string>차트 열기</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
